src/info.cpp

```diff
@@ -35,16 +38,20 @@ void read_char_info(FINGER_INFO *info);
         break;                                                                                                         \
     }
 
-/* For finger info */
-FINGER_INFO *info_cache = nullptr;
+// For finger info.
+namespace {
```

Comment:
I notice you've namespaced these new sections of act_info.cpp like here and down on line 76, wondering if they can be consolidated together

Reply:
Can do: I was (for now) sticking to keeping all the heinous globals bunched up together at the top. I think it's cool to move them in this case, there aren't too many. Also I don't actually know if `player_list` is used meaningfully anywhere...I'll add a TODO and issue to track that